### DISCUSSION TITLE
Assistance request - Load exported CSV

### DISCUSSION THREAD

**Discussion by weimdall** [firefly-iii/firefly-iii#4507] [Firefly III]
I am running Firefly III version 5.4.6.

**Description**
My Firefly burnt away few days ago, it was hosted in the OVH datacenter that has been destroyed. ( [News](https://www.datacenterknowledge.com/uptime/fire-has-destroyed-ovh-s-strasbourg-data-center-sbg2) )
I just got confirmation that none of my data are recoverable.

As I thought a datacenter was a safe place to store data, I didn't really had a complete backup procedure ..
Now I have to start a new firefly .. but ""luckily"" I have a CSV from 5 month ago that I got using the "Export Data" feature.
It contains only the transactions, but it's a start (I have 5 years of transactions in that CSV).

I tried to import it .. but I got quite confused
I remember a functionality within firefly to import csv, it seems that it's no longer available ?
So I tried using the "csv-importer", but it asked me for a JSON file with the CSV.

What is the best way to load my CSV ? 
And is there an available JSON file that can be used with the CSV downloaded through "Export Data" ?

**Comment by JC5** [comment]
That sucks, sorry to hear that :(

The JSON file is optional.

https://docs.firefly-iii.org/csv/usage/upload/

**Reply by weimdall** [reply]
Oh, all right 
I was using the CLI instead of the Web UI
Before importing them, I had to create the bills and budgets (not automatically created).

I have sill one light issue, it created me revenue accounts "Reconciliation" and "Initial balance" for each asset account.
So, I moved them as "Inactive"..is there something I can do in the BDD about it ?

**Reply by JC5** [reply]
In the BDD?

**Reply by weimdall** [reply]
DB - Database , sorry 😅

**Reply by JC5** [reply]
Ah check! Yes, you can find them and delete them, but otherwise they won't do a lot of harm.
